strings/base_activation.h

```diff
@@ -18,8 +18,11 @@ namespace winrt::impl
 
     using library_handle = handle_type<library_traits>;
 
-    template <bool isSameInterfaceAsIActivationFactory>
-    WINRT_IMPL_NOINLINE hresult get_runtime_activation_factory_impl(param::hstring const& name, winrt::guid const& guid, void** result) noexcept
+    // This function pointer will be null unless one or more translation units in a binary define WINRT_REG_FREE before
+    // including winrt/base.h.  If that is defined then the overall binary will support regfree COM activation.
+    inline hresult(*reg_free_factory_getter)(param::hstring const&, guid const&, void**) = nullptr;
+
+    WINRT_IMPL_NOINLINE inline hresult get_runtime_activation_factory_impl(param::hstring const& name, winrt::guid const& guid, void** result) noexcept
```

Comment:
Leave the template on get_runtime_activation_factory_impl. Just remove the isSameInterfaceAsIActivationFactory parameter when calling the reg_free_factory_getter

Reply:
Oh wait, if we do that, then there's nothing in get_runtime_activation_factory_impl to template on. Gotcha.